dev-python/txtorcon/txtorcon-21.1.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{8,9,10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Twisted-based Tor controller client, with state-tracking and config abstractions"
+HOMEPAGE="https://github.com/meejah/txtorcon https://pypi.org/project/txtorcon/ https://txtorcon.readthedocs.org"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/automat[${PYTHON_USEDEP}]
+	dev-python/cryptography[${PYTHON_USEDEP}]
+	dev-python/incremental[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-python/twisted-16.0.0[${PYTHON_USEDEP},crypt]
+	>=dev-python/zope-interface-3.6.1[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/automat[${PYTHON_USEDEP}]
+		dev-python/sphinx[${PYTHON_USEDEP}]
+		dev-python/repoze-sphinx-autointerface[${PYTHON_USEDEP}]
+		>=dev-python/zope-interface-3.6.1[${PYTHON_USEDEP}]
+	)
+"
+
+DEPEND="
+	${RDEPEND}
```

Comment:
FYI: we usually don't need to do DEPEND="${RDEPEND}" (or BDEPEND=...) for python stuff, it usually only needs it at runtime, but if it checks at build time, i guess it's worth just requring it